packages/create-sitecore-jss/src/templates/nextjs-sxa/src/components/Navigation.tsx

```diff
@@ -134,8 +135,8 @@ const NavigationList = (props: NavigationProps) => {
       >
         <Link
           field={getLinkField(props)}
-          editable={sitecoreContext.pageEditing}
           onClick={props.handleClick}
+          {...(!isMetadataMode ? { editable: sitecoreContext.pageEditing } : '')}
```

Comment:
What does metadata editMode have to do with this? This particular `editable` is an attribute on the `Link` field-helper React component (different than `editable` on a layout field). Should be able to keep it as it was.